### PR TITLE
Added max_age helper.

### PR DESCRIPTION
**Pull request by hex337**
- Also updated tests to use expect format
- Updated the Keen gem version requirement

### CODE REVIEW

**Comment by hex337** [comment]
@josephwegner Can you take a look at this when you have time?

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'pushpop'
-gem 'keen'
+gem 'keen', '>=0.9.2'
```
Comment (josephwegner) [inline Gemfile]: want to do `~0.9`? I think that gets us all non-breaking changes.

**Comment by josephwegner** [comment]
+1 :ship: :shipit: :pizza: :banana:

File: Gemfile (hunk 1 above)
Comment (hex337) [inline Gemfile]: 0.9.2 adds specific support for max_age as an integer, so I wanted to make sure it was at least that version.

Comment (josephwegner) [inline Gemfile]: +1

**Pull request merged**